### ISSUE TITLE
Menu overflow with many applications

### ISSUE THREAD

**Issue by probonopd** [rodlie/draco#32]
OK, my use case may be a little extreme, but when one has too many applications to fit into the menu there seems to be no obvious way to scroll further to the right...

![](https://user-images.githubusercontent.com/2480569/81448313-b4436580-916d-11ea-8756-33a30e407949.png)

**Comment by rodlie** [comment]
Yeah, never tested the menu with that many items ;) Will take a look into that.
